.github/workflows/main.yml

```diff
@@ -7,12 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Use Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
-          node-version: '12.x'
+          node-version: 18
       - run: npm install
       - run: npm run build --if-present
```

Comment:
Thanks for the PR. Looks like the check is failing. Seems to be fixed by providing `NODE_OPTIONS=--openssl-legacy-provider` option

```
NODE_OPTIONS=--openssl-legacy-provider npm run build --if-present
```

Reply:
@agenthunt added you as a collaborator to my fork so you can wrap this up.

Reply:
Thanks. Done and Released v2.0.0